src/_scss/pages/homepageUpdate/_wordOfTheDay.scss

```diff
@@ -104,5 +107,24 @@
 
   .word-of-the-day__error-message {
     text-align: center;
+    color: $gray-2;
+  }
+  .word-of-the-day-paragraph-one{
+    font-size: rem(24);
+    text-align: center;
+    font-weight: $font-semibold;
+    margin-bottom: rem(4);
+  }
+  .word-of-the-day-paragraph-two{
+    font-size: rem(16);
+    margin-top: rem(4);
+  }
+  .word-of-the-day-button{
+    border-radius: rem(4);
+    border: solid rem(1) #dfe1e2;
```

Comment:
color variables from _scss/cores/variables.scss could be used here for the border and background-color

Reply:
Changed to color  var